sky/services/semantics/semantics.mojom

```diff
@@ -53,7 +53,7 @@ struct SemanticGeometry {
 };
 
 [ServiceName="semantics::SemanticsClient"]
```

Comment:
You should probably just remove the ServiceName.  I don't think you intend someone to provide this interface via a ServiceProvider.